build.gradle

```diff
@@ -229,6 +229,8 @@ project(':ambry-admin') {
         testCompile project(':ambry-clustermap').sourceSets.test.output
         testCompile project(':ambry-rest').sourceSets.test.output
         testCompile project(':ambry-utils').sourceSets.test.output
+        testCompile project(':ambry-network').sourceSets.test.output
+        testCompile "org.bouncycastle:bcpkix-jdk15on:$bouncycastleVersion"
```

Comment:
This dependency is used to generate self signed certificates in `TestSSLUtils`. I guess I need to explicitly include it for it to be resolved in other modules.